scripts/docgen.lua

```diff
@@ -76,7 +76,7 @@ require'nvim_lsp'.{{template_name}}.setup{}
 
 local function make_lsp_sections()
   return make_section(0, '\n', sorted_map_table(skeleton, function(template_name, template_object)
-    local template_def = template_object.template_config
+    local template_def = template_object.document_template
```

Comment:
not sure this should change

Reply:
I was trying to remove references to config that weren't directly related to the configuring the server/client. I'll revert.